sql/analyze.go

```diff
@@ -1222,8 +1222,7 @@ func simplifyComparisonExpr(n *parser.ComparisonExpr) parser.Expr {
 			return n
 		case parser.In, parser.NotIn:
 			tuple := n.Right.(parser.DTuple)
-			sort.Sort(tuple)
-			tuple = uniqTuple(tuple)
+			tuple.Normalize()
```

Comment:
Is this necessary? Normalization should already have been performed before `analyzeExpr` is called.